src/ui/TextMessageItemBody/index.tsx

```diff
@@ -40,7 +40,7 @@ export default function TextMessageItemBody({
     return tokenizeMessage({
       messageText: message?.message,
     });
-  }, [message?.updatedAt]);
+  }, [message]);
```

Comment:
@ishubham21 Thanks for your contribution and the detailed explanation. I see your point. 

```suggestion
  }, [message?.message]);
```
I wonder ^ this would this fix your issue too. I think this value will be updated when the streaming message is coming. 
The reason I'm asking is, putting the whole object in the dependency array is not an optimal way in terms of performance.

Reply:
@AhyoungRyu 
Definitely this will fix the issue we are facing. This PR was just an indicate solution - the root cause lies with `updatedAt` not being set properly. 
If we proceed with this change, can we also do this for scroll events as well? I believe they too have their dependencies set as `updatedAt`

Reply:
Yeah if you're talking about [this part](https://github.com/sendbird/sendbird-uikit-react/blob/main/src/modules/Channel/components/Message/index.tsx#L152), will make the change as well.

Reply:
@AhyoungRyu yes that’s the part. Thank you for pointing it out. Should I push the update, or sendbird team will do it?

Reply:
Oh I'll do that. Thanks for confirming it :)

Reply:
@ishubham21 Since we're not accepting the code change from outside at the moment, I made a separate PR based on yours. https://github.com/sendbird/sendbird-uikit-react/pull/928 
We're aiming to publish a new version this afternoon. We'll let you know once it's out. 

Thanks again :)